AppDevAnnouncements/Classes/Endpoint.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  Endpoint.swift
+//  AppDevAnnouncements
+//
+//  Created by Gonzalo Gonzalez on 11/9/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import Foundation
+import FutureNova
+
+extension Endpoint {
+
+    static func setupEndpointConfig(_ scheme: String, _ host: String, _ commonPath: String) {
+        Endpoint.config.scheme = scheme
+        //Endpoint.config.port =
```

Comment:
delete

Reply:
i left this because i wasn't sure if we would need it for the backend; i'll confirm with Sean if necessary